### PR TITLE
Add Fantom to Keepers Supported Networks

### PR DESCRIPTION
**Pull request by ritika-b**
- add Fantom config details to Supported Networks page
- add Fantom to overview

### CODE REVIEW

File: docs/chainlink-keepers/supported-networks.html [hunk 1]
```diff
@@ -33,6 +33,9 @@
 - Avalanche:
   - Mainnet
   - Fuji Testnet
+- Fantom:
+  - Testnet
+  - Mainnet
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:38]:
```suggestion
- Fantom:
  - Mainnet
  - Testnet
```

File: docs/chainlink-keepers/supported-networks.html [hunk 2]
```diff
@@ -93,5 +96,21 @@
 
 {% endmarkdown %}
 
+{% markdown %}
+### Fantom Mainnet
+
+* **Payment Premium (paymentPremiumPPB):** <span>50</span>% gas premium
+    This premium is applied on top of the gas cost paid by the Keeper to perform your Upkeep
+* **Block Count Per Turn (blockCountPerTurn):** <span>200</span>
```
Comment (dwightjl) [inline docs/chainlink-keepers/supported-networks.html:104]: The contract seems to indicate this should be 20. Let's confirm.

Comment (wentzeld) [reply docs/chainlink-keepers/supported-networks.html:104]: 200 block count per turn is good, thanks for double checking. I just updated contract to match final intended config.l

Comment (wentzeld) [reply docs/chainlink-keepers/supported-networks.html:104]: 200 block count per turn is good, thanks for double checking. I just updated contract to match final intended config.

File: docs/chainlink-keepers/supported-networks.html (hunk 1 above)
Comment (wentzeld) [reply docs/chainlink-keepers/supported-networks.html:38]: Thanks

**Comment by wentzeld** [comment]
Looks good to me, thanks for speedy turn around.

**Review by dwightjl — APPROVED** [review]

**Pull request merged**